mutalyzer/website/templates/description-extractor.html

```diff
@@ -29,16 +39,57 @@
 Please supply a reference sequence and an observed sequence.
 </p>
 
-<form action="{{ url_for('.description_extractor') }}" method="get" class="form">
-  <div class="form-group">
-    <label for="reference_sequence">Reference sequence</label>
-    <input type="text" name="reference_sequence" id="reference_sequence" value="{{ reference_sequence }}" class="form-control form-pre example-target" placeholder="Reference sequence">
-    <p>Example: <code class="example-input">ATGATGATCAGATACAGTGTGATACAGGTAGTTAGACAA</code></p>
-  </div>
-  <div class="form-group">
-    <label for="variant_sequence">Observed sequence</label>
-    <input type="text" name="variant_sequence" id="variant_sequence" value="{{ variant_sequence }}" class="form-control form-pre example-target-2" placeholder="Observed sequence">
-    <p>Example: <code class="example-input-2">ATGATTTGATCAGATACATGTGATACCGGTAGTTAGGACAA</code></p>
+<form action="{{ url_for('.description_extractor') }}" method="get" class="form" id="invoer">
+  <div class="row">
+    <div class="col-md-6">
+      <div class="form-group" id="input-methods">
+        <div class="radio">
+          <label>
+            <input type="radio" name="method" value="raw_method" {{ 'checked' if method == 'raw_method' or not method }}>
+            Provide two raw sequences.
+          </label>
+        </div>
+        <div class="radio">
+          <label>
+            <input type="radio" name="method" value="refseq_method" {{ 'checked' if method == 'refseq_method' }}>
+            Provide two RefSeq accession numbers.
+          </label>
+        </div>
+      </div>
+    </div>
+
+    <script type="text/javascript">
+    $('#input-methods input').on('change', updateVisibility);
+    </script>
+
+    <div class="col-md-6">
+      <div class="subform" id="raw_method">
+        <div class="form-group">
+          <label for="reference_sequence">Reference sequence</label>
+          <input type="text" name="reference_sequence" id="reference_sequence" value="{{ reference_sequence }}" class="form-control form-pre example-target" placeholder="Reference sequence">
+          <p>Example: <code class="example-input">ATGATGATCAGATACAGTGTGATACAGGTAGTTAGACAA</code></p>
```

Comment:
Indeed, this is the relevant code from `mutalyzer/website/templates/static/js/interface.js`:

```javascript
$('.example-input').on('click', function() {
  $('.example-target').val($(this).text());
  return false;
});
```

This is a bit brittle as the target element is implicit by the class name. It also doesn't scale well in the number of input fields.

I suggest to change it to something like:

```javascript
$('.example-input').on('click', function() {
  var target = document.getElementById($(this).data('for'));
  $(target).val($(this).text());
  return false;
});
```

Where we make the target element explicit in HTML with:

```html
<code class="example-input" data-for="reference_sequence">ATGATGATC</code>
```